manager/assets/modext/widgets/system/modx.grid.context.js

```diff
@@ -46,7 +46,7 @@ MODx.grid.Context = function(config) {
             ,editor: { xtype: 'numberfield' }
         }]
         ,tbar: [{
-            text: _('create_new')
+            text: _('context_create')
```

Comment:
You can't rename the lexicon key here without renaming it in the lexicon files.

Reply:
This entry is on the way `core/lexicon/en/context.inc.php` - in 11 line. But she was not used